### PR TITLE
jupyter-health: remove profile list

### PR DESCRIPTION
**Pull request by minrk**
for smoother launch, since we only have one image

promotes default profile to sole singleuser configuration, unmodified

### CODE REVIEW

File: config/clusters/jupyter-health/common.values.yaml [hunk 1]
```diff
@@ -53,69 +53,15 @@ jupyterhub:
...
-            choices: {}
-          resource_allocation: *profile_options_resource_allocation
+    image: quay.io/jupyterhealth/singleuser-premvp:latest
+    memory:
+      limit: 1991244775
+      guarantee: 1991244775
+    cpu:
+      limit: 3.725
+      guarantee: 0.2328125
+    node_selector:
```
Comment (consideRatio) [inline config/clusters/jupyter-health/common.values.yaml:63]:
```suggestion
    nodeSelector:
```

**Comment by minrk** [comment]
Thanks!

**Comment by minrk** [comment]
Launches are failing now, presumably a typo somewhere here?

**Comment by minrk** [comment]
spawn is failing with: `No such profile: pangeo`

**Comment by consideRatio** [comment]
@minrk I think what happens is that we have `user_options` state saved in the jupyterhub database about a profile slug, and when kubespawner receives it and there isn't a profile - it errors. I believe a fix could be to reset the jupyterhub DB for example. Do you want me to apply that fix short term?

- should jupyterhub retain and sending old user_options?
- should kubespawner handle this in a more fault tolerant way?

I think this is an upstream issue overall, but I'm not sure where.

**Comment by consideRatio** [comment]
You get the same error if you use an incognito window right - because there isn't any user_option state in the frontend etc?

**Comment by consideRatio** [comment]
@minrk I went ahead with what I think is the fix - hub-db-dir cleared and re-deployed

**Comment by minrk** [comment]
Thanks! Works now. Something to think about in either kubespawner or JupyterHub: what happens when profilelist config is removed after being in use. Simplest version in kubespawner would be to ignore profile `if not profile_list`. Could be with a warning.

**Pull request merged**